### DISCUSSION TITLE
How to compile examples in VS2019?

### DISCUSSION THREAD

**Discussion by ZedZipDev** [spectreconsole/spectre.console#802] [Q&A] [answered]
I have tried to build it in VS2019 but received a lot of errors
No LangVersion 10

Is it possible to compile it in VS2019?

**Comment by patriksvensson** [comment, marked as answer]
@Oleg26Dev No, you will need VS2022 to compile this since we use C# 10.

**Reply by ZedZipDev** [reply]
I see. Ok. 
But can I create a cross-platform app (NET 5)in VS2019  using Spectre.Console NuGet?

**Reply by patriksvensson** [reply]
@Oleg26Dev Yes, Spectre.Console is available for NET 5.
Version 1.0 however will not since .NET 5 is out of support in two months.

**Reply by ZedZipDev** [reply]
Ok, thank you.
